### PR TITLE
Moved `micro` to `peerDependencies`

### LINKED ISSUE #31: micro as a peerDependency
Hi everyone, how are you?

What do you think about define `micro` as a peerDependency?

For example, in my case I'm using micro 8.0.4 in production but when I run micro-dev I have two differents intances of `micro` working on (8.0.4 and 8.0.2) because how NPM resolve my dependencies.

This is a problem when `micro` uses some kind of internal state behaviour like this one: https://github.com/zeit/micro/blob/master/lib/index.js#L114

### PR DESCRIPTION
**Pull request by fmiras**
According to #31 this PR uses micro of the main project.

This closes #78 and fixes #31.

### CODE REVIEW

**Comment by leo** [comment]
Sounds like a great idea! But we shouldn't hardcode the name of `node_modules` anywhere, in my opinion. We need to use a property of `require.` to find the module and load it.

File: lib/log.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+const { join } = require('path')
+
 // Packages
 const chalk = require('chalk')
 const jsome = require('jsome')
 const PrettyError = require('pretty-error')
 const stringLength = require('string-length')
-const { json, send } = require('micro')
+
+const { json, send } = require(join(process.cwd(), 'node_modules/micro'))
```
Comment (timneutkens) [inline lib/log.js]: You can just require micro. There’s no need to change this. Same Goes for the other cases. Also add micro to the devdeps 👍🏻

**Review by timneutkens — DISMISSED** [review]

**Comment by fmiras** [comment]
Sorry, I hate peerDependencies 😆.

**Comment by sijad** [comment]
any news?

**Review by leo — APPROVED** [review]

**Pull request merged**